tools/autograd/derivatives.yaml

```diff
@@ -1662,23 +1662,23 @@
 - name: _to_dense(Tensor self, ScalarType? dtype=None) -> Tensor
   self: to_dense_backward(grad, self)
 
-- name: to_sparse(Tensor self, *, Layout? layout=None, int[2]? blocksize=None) -> Tensor
-  self: to_sparse_backward(grad, self.layout(), self.sym_blocksize())
+- name: to_sparse(Tensor self, *, Layout? layout=None, int[2]? blocksize=None, int n_dense_dim=0) -> Tensor
+  self: to_sparse_backward(grad, self.layout(), self.sym_blocksize(), n_dense_dim)
```

Comment:
I am not sure this is correct. The backward function need to receive `n_dense_dim = self.n_dense_dim`, not that of the forward. This means that we need to write an autogen replace for `n_dense_dim` similar to how it is done for the blocksize with `sym_blocksize`.

Reply:
Could you please check the latest update?  I think it should do it, except that I'm not sure is there something better than `n_dense_dim.as_int_unchecked()` in `to_sparse_backward()` method (in FunctionsManual.cpp).

As far as tests you mentioned: this n_dense_dim argument is for now used only for dense to sparse conversions, in which case the reverse should be conversion to dense, so these are not relevant for now.